optimade/validator/utils.py

```diff
@@ -36,6 +36,9 @@
     StructureResource,
 )
 
+DEFAULT_CONN_TIMEOUT = 0.5
```

Comment:
Could we make this 5 s instead? Especially the Materials Cloud implementations is slow, and the time out is merely a maximum time, so if the request is eventually faster that's the run time...
```suggestion
DEFAULT_CONN_TIMEOUT = 5
```

Reply:
This is only the time for it to acknowledge connection, not the time to complete the request. I've found that this works even for other very slow implementations.

As each request is repeated 5 times on connection failure, having 5 seconds feels like overkill.

Reply:
Hmm, according to https://docs.python-requests.org/en/latest/user/advanced/#timeouts we should actually set this to 3*n*+ε for integer *n* which is the default TCP packet retransmission window, though that is hardening against a poor connection rather than a slow server. Would that be okay with you @CasperWA?

Reply:
fine by me. I think usually the requests to the materials cloud servers take some seconds to return anything with certain filters. That's just my experience - but if you say that's fine, then cool :)

Reply:
I think the real time is obfuscated considerably by whatever client you use (curl, browser etc) which will only "do something" once the entire response has come in. The TCP handshake (I think I am using this terminology correctly) should be quick provided the server exists. I'll bump it to 3.05 seconds anyway.